core/shared/src/main/scala/zio/internal/FiberRuntime.scala

```diff
@@ -37,14 +37,15 @@ final class FiberRuntime[E, A](fiberId: FiberId.Runtime, fiberRefs0: FiberRefs,
   import FiberRuntime.{AsyncJump, DisableAssertions, EvaluationSignal, stackTraceBuilderPool}
   import ZIO._
 
-  private var _lastTrace      = fiberId.location
-  private var _fiberRefs      = fiberRefs0
-  private var _runtimeFlags   = runtimeFlags0
-  private var _blockingOn     = FiberRuntime.notBlockingOn
-  private var _asyncContWith  = null.asInstanceOf[ZIO.Erased => Any]
-  private val running         = new AtomicBoolean(false)
-  private val inbox           = new ConcurrentLinkedQueue[FiberMessage]()
-  private var _children       = null.asInstanceOf[JavaSet[Fiber.Runtime[_, _]]]
+  private var _lastTrace     = fiberId.location
+  private var _fiberRefs     = fiberRefs0
+  private var _runtimeFlags  = runtimeFlags0
+  private var _blockingOn    = FiberRuntime.notBlockingOn
+  private var _asyncContWith = null.asInstanceOf[ZIO.Erased => Any]
+  private val running        = new AtomicBoolean(false)
+  private val inbox          = new ConcurrentLinkedQueue[FiberMessage]()
+  private var _children      = null.asInstanceOf[JavaSet[Fiber.Runtime[_, _]]]
+  @volatile
```

Comment:
Sorry for the nitpick, but for consistency, you inline the annotation with the var definition? Also perhaps move it to the line above the `_exitValue` so that volatile vars are grouped together